### DISCUSSION TITLE
"type" as column name

### DISCUSSION THREAD

**Discussion by berma83** [Budibase/budibase#2939] [Help]
Hi!

Column named as "type" is automatically filled with "row". Is this an expected behavior?

Evidence:

https://user-images.githubusercontent.com/53205779/136608518-9487405a-a0aa-4b52-b376-7919f40c711d.mp4

**Comment by Pachat** [comment]
May be  [this](https://stackoverflow.com/a/58461946) gives an insight.

**Comment by mjashanks** [comment]
Thanks for reporting this - we should disallow the name `type`, as it's used internally. You can use `Type` if you like!

Logged this here: https://github.com/Budibase/budibase/issues/2958
